std-bits/table/src/main/java/org/enso/table/parsing/BooleanParser.java

```diff
@@ -0,0 +1,42 @@
+package org.enso.table.parsing;
+
+import org.enso.table.data.column.builder.object.BoolBuilder;
+import org.enso.table.data.column.builder.object.Builder;
+import org.enso.table.parsing.problems.InvalidFormatProblemAggregator;
+
+public class BooleanParser extends TypeParser<InvalidFormatProblemAggregator> {
+
+  private final String[] trueValues;
```

Comment:
True, but this will stop working when we start using unicode-normalizing comparisons.

But for now it can work, so I'll use set and we can easily revert that change if necessary in the future.

Reply:
We could use the graalvm EconomicSet with a custom Equivalence when we want to sort that out. 

Alternatively, in this specific case if we normalise the values first, then the set would still work as we aren't allowing for case insensitive operations.

Reply:
Fair point